index.js

```diff
@@ -32,8 +32,19 @@ app.get('/todos', (request, response) => {
   });
 });
 
+// Update the title via server:port/todos/id/new_title
+app.patch('/todos/:id/:title', (request, response) => {
```

Comment:
If I can chip in here, these are two lovely learning cases: @majdiali @beto0o0o please also read this.

1) About `todos` (plural) or `todo` (singular): this is a very common issue many teams run into. The rule is to always make all url's either plural or singular. Don't to both. In this case, we already started with `todos` so I would suggest to stick with that. Also, remember that the `/:id` gives an indication that it's one specific todo